src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -125,9 +126,11 @@ private function buildImportMapJson(): void
             return;
         }
 
-        $dumpedPath = $this->assetMapper->getPublicAssetsFilesystemPath().'/'.self::IMPORT_MAP_FILE_NAME;
-        if (file_exists($dumpedPath)) {
-            $this->json = file_get_contents($dumpedPath);
+        $dumpedImportMapPath = $this->assetMapper->getPublicAssetsFilesystemPath().'/'.self::IMPORT_MAP_FILE_NAME;
+        $dumpedModulePreloadPath = $this->assetMapper->getPublicAssetsFilesystemPath().'/'.self::IMPORT_MAP_PRELOAD_FILE_NAME;
+        if (file_exists($dumpedImportMapPath) && file_exists($dumpedModulePreloadPath)) {
```

Comment:
we prefer `is_file` over `file_exists` when possible because it's been proved faster (even if we don't really care here, but that's a CS thing now)